domain/modelconfig/validators/validators.go

```diff
@@ -164,46 +164,3 @@ func LoggingTracePermissionChecker(canTrace bool) config.ValidatorFunc {
 		return cfg, nil
 	}
 }
-
```

Comment:
We are still going to need a validator that checks the user isn't trying to set the secret backend via model config. We can't control it as client only logic. We don't control out clients in every case.

Reply:
We are going to follow this up in a seperate PR with a validator that errors out for any unknown model config keys.

@tlm will be circulating this decision for further feedback.